docs/modules/hdfs/pages/usage-guide/operations/pod_disruptions.adoc

```diff
@@ -0,0 +1,108 @@
+
+= Pod disruptions
+
+You can configure Pod disruptions for HDFS nodes as described in xref:concepts:operations/pod_disruptions.adoc[].
+
+Unless you configure something else or disable our PodDisruptionBudgets (PDBs), we write the following PDBs:
```

Comment:
```suggestion
Unless you configure something else or disable our `PodDisruptionBudgets` (PDBs), we write the following PDBs:
```

Reply:
@fhennig I think our guidelines in https://docs.stackable.tech/home/stable/contributor/style_guide tell us to not format k8s objects, is that correct?

> [Use PascalCase for API objects](https://kubernetes.io/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects) such as ConfigMap or KafkaCluster

Reply:
Yes